runners/spark/src/main/java/org/apache/beam/runners/spark/util/BroadcastHelper.java

```diff
@@ -77,8 +77,8 @@ public synchronized T getValue() {
     }
 
     @Override
-    public void broadcast(JavaSparkContext jsc) {
-      this.bcast = jsc.broadcast(value);
+    public synchronized void broadcast(JavaSparkContext jsc) {
+      this.bcast = jsc.broadcast(getValue());
```

Comment:
This is generally awkward...
The entire helper is written in a way that guarantees (constructor) that a `value` exists in the driver before broadcasting, while it's not serialized and sent to the workers (`transient T value`) but the workers are guaranteed to have a broadcast available.
For the way `Broadcast` variables work, findbugs analysis is just wrong.
I think that all "issues" in this class can be ignored on account of `Broadcast` variables being a "Spark-specific" distributed implementation.